alma/lib/Model/ShippingData.php

```diff
@@ -117,10 +117,16 @@ public static function shippingInfo($cart)
      */
     private static function shippingInfoData($carrier, $carrierInfo)
     {
+        $title = $carrier->delay;
```

Comment:
you can also write it like 
```
		$title = (is_array($carrier->delay)) ? implode(', ', $carrier->delay) : $carrier->delay;

```

Reply:
@ClaireAlma I put him directly in the array. thanks